Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -663,3 +663,47 @@ def test_update_remote_data_sc_task(mocker):
     mocker.patch('ServiceNowv2.get_ticket_fields', side_effect=ticket_fields)
     mocker.patch.object(client, 'update', side_effect=update_ticket)
     update_remote_system_command(client, args, params)
+
+
+def test_handle_ampersands_in_query():
```

Comment:
add more cases + edge cases

Reply:
The only other one I can think of is a recursive '&' usage like "banana=3&user=admin=true&id=5" but I don't know if requests.requests supports it nor if there is a use/support for it in ServiceNow queries. Also it is out of scope of the bug and I'm not sure its worth the extra effort. 

If you have another use case in mind that I didn't think of i'd be happy to hear